### PR TITLE
Fix build include xtensor headers

### PR DESCRIPTION
**Pull request by konradkaras**
PR fixes missing xtensor headers when building

### CODE REVIEW

**Comment by roelaaij** [comment]
Did you have `xtensor` installed system wide or was it using the version being built in-place?

**Comment by konradkaras** [comment]
It was the built-in version

File: CMakeLists.txt [hunk 1]
```diff
@@ -168,6 +168,11 @@ if(ENABLE_PYTHON)
     target_compile_definitions(k40gen PRIVATE "-DXTENSOR_ENABLE_XSIMD")
   endif()
 
+  # xtensor headers
+  find_package(xtensor REQUIRED)
```
Comment (roelaaij) [inline CMakeLists.txt]: Does it also work without this line?

File: CMakeLists.txt [hunk 2]
```diff
@@ -168,6 +168,11 @@ if(ENABLE_PYTHON)
...
+  target_include_directories(k40gen PUBLIC ${xtensor_INCLUDE_DIRS})
+  target_link_libraries(k40gen PUBLIC xtensor)
```
Comment (roelaaij) [inline CMakeLists.txt]: I think xtensor has no libraries, so is this one needed?

File: CMakeLists.txt (hunk 1 above)
Comment (konradkaras) [reply CMakeLists.txt]: yes, it works without that line, I'll update it

File: CMakeLists.txt (hunk 2 above)
Comment (konradkaras) [reply CMakeLists.txt]: indeed, it's not needed here

**Pull request merged**